src/main/java/TaskList.java

```diff
@@ -0,0 +1,43 @@
+public class TaskList {
+    private Task[] tasks = new Task[100];
+    private int taskCount = 0;
+
+    public int getTaskCount(){
+        return taskCount;
+    }
+
```

Comment:
spacing in between methods could be more consistent , makes the code longer than it has to be